float64[3] target  # x, y, theta
---
bool success
string message